daisy_workflows/image_import/suse/translate.py

```diff
@@ -26,67 +26,159 @@
 import utils.diskutils as diskutils
 
 
-network = '''
-BOOTPROTO='dhcp'
-STARTMODE='auto'
-DHCLIENT_SET_HOSTNAME='yes'
-'''
+class Package:
+  name = ''
+  gce = False
+  required = False
+
+  def __init__(self, name, gce, required):
+    self.name = name
+    self.gce = gce
+    self.required = required
+
+
+class SuseRelease:
+  flavor = ''
+  major = ''
+  minor = ''
+  packages = None
+  subscriptions = None
+
+  def __init__(self, flavor, major, minor, packages, subscriptions=None):
+    self.flavor = flavor
+    self.major = major
+    self.minor = minor
+    self.packages = packages
+    self.subscriptions = subscriptions
+
+
+cloud_init = Package('cloud-init', gce=False, required=False)
+gcp_sdk = Package('google-cloud-sdk', gce=True, required=False)
+gce_init = Package('google-compute-engine-init', gce=True, required=True)
+gce_oslogin = Package('google-compute-engine-oslogin', gce=True, required=True)
+
+_distros = [
+    SuseRelease(
+        flavor='opensuse',
+        major='15',
+        minor='1',
+        packages=[cloud_init, gcp_sdk, gce_init, gce_oslogin]
```

Comment:
packages are initialized to the same list for all 3 distros. Doesn't seem that it should be an argument as it's the same for all 3.

Reply:
These fields are for supporting GCP licensing and the SAP variant of SLES, both of which will need some additional packages. Check out the zypper command on this page:

https://www.suse.com/c/create-sles-demand-images-gce/

I considered having a 'base' set of common packages, but in my experience that is confusing for the reader, and creates a large blast radius when someone updates the base set.

Reply:
So this is for future proofing?

Reply:
went ahead and refactored the list of packages out of the field and into a variable.